package-lock.json

```diff
@@ -7327,9 +6970,9 @@
       }
     },
     "node_modules/@mongodb-js/compass-editor": {
-      "version": "0.36.5",
-      "resolved": "https://registry.npmjs.org/@mongodb-js/compass-editor/-/compass-editor-0.36.5.tgz",
-      "integrity": "sha512-9JxwWR1wTfRFfLM90j/KnNlQYxjmSpQq4nUm2pU+W/2B7UWu+ILifCrYs8+9WlHiZSgDnD1qMwo82nOyKrpWag==",
+      "version": "0.36.4",
```

Comment:
This seems like a downgrade - did npm install mess something up?

Reply:
yeah something weird happened with `npm i` here